### PR TITLE
simplest thing that could work

### PR DESCRIPTION
**Pull request by nelz9999**
solves #15

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -226,6 +229,31 @@ func addEnvVars(ciProvider string) {
...
+	data, err := ioutil.ReadFile(locn)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "unable to read %q: %v\n", locn, err)
+		return
+	}
+
+	err = logfmt.Unmarshal(
+		data,
+		logfmt.HandlerFunc(func(key, val []byte) error {
+			libhoney.AddField(string(key), string(val))
```
Comment (maplebed) [inline main.go]: Given one known expected use case is adding the size of generated artifacts, an attempt to coerce the values to numbers would be a kindness. 

```suggestion
			if f, err := strconv.ParseFloat(string(val), 64); err == nil {
				libhoney.AddField(string(key), f)
				return nil
			}
			if b, err := strconv.ParseBool("true"); err == nil {
				libhoney.AddField(string(key), b)
				return nil
			}
			libhoney.AddField(string(key), string(val))
```

File: main.go [hunk 2]
```diff
@@ -226,6 +229,31 @@ func addEnvVars(ciProvider string) {
...
+			return nil
```
Comment (maplebed) [inline main.go]: should just be `return`

Comment (nelz9999) [reply main.go]: Hrm? The func signature specifies an error return value, no?

Comment (maplebed) [reply main.go]: derp, I was looking at the return signature for `addfields()` not the handler function.  Nothing to see here...

File: main.go [hunk 3]
```diff
@@ -226,6 +229,39 @@ func addEnvVars(ciProvider string) {
...
+	}
+
+	err = logfmt.Unmarshal(
+		data,
+		logfmt.HandlerFunc(func(key, val []byte) error {
+			if f, err := strconv.ParseFloat(string(val), 64); err == nil {
+				libhoney.AddField(string(key), f)
+				return nil
+			}
+			if b, err := strconv.ParseBool("true"); err == nil {
```
Comment (maplebed) [inline main.go]: 🤦‍♂ sorry I gave you a bad suggestion before.

```suggestion
			if b, err := strconv.ParseBool(string(val)); err == nil {
```

Comment (nelz9999) [reply main.go]: Also on me... I shoulda looked more carefully before mashing the "commit" button!

**Pull request merged**